add-on/chrome/bootstrap.js

```diff
@@ -187,15 +187,8 @@ var WindowListener = {
             this.LoopAPI.initialize();
 
             let anchor = event ? event.target : this.toolbarButton.anchor;
-            let setHeight = 410;
-            if (gBrowser.selectedBrowser.getAttribute("remote") === "true") {
-              setHeight = 262;
-            }
             this.PanelFrame.showPopup(window, anchor,
-              "loop", null, "about:looppanel",
-              // Loop wants a fixed size for the panel. This also stops it dynamically resizing.
-              { width: 330, height: setHeight },
-              callback);
+              "loop", null, "about:looppanel", null, callback);
```

Comment:
While you're here, can make this one param per line with // comment by each line describing what it is to make this less error-prone to maintain?